Makefile

```diff
@@ -44,6 +44,11 @@ test-integration: ## Runs integration tests with phpunit/phpunit
 test-docs: ## Generate projects docs without warnings
 	$(PHP_BIN) vendor/bin/guides -vvv --no-progress Documentation --output="/tmp/test" --fail-on-log
 
+.PHONY: test-xml
+test-xml: ## Lint all guides.xml
+	./tools/xmllint.sh
```

Comment:
Please add this to the QA workflow on github, so this is validated before we can merge.

Reply:
Can we do that? It is pure linux?

Reply:
Yes, github actions are executed on a full Ubuntu stack.

Reply:
would I just call the script then somehow in the github actions?

Reply:
You can call this script in the workflow like this:

```
      - name: "Lint xml configs"
        run: "tools/xmllint.sh"
```

Reply:
added it, please have a look